templates/login.php

```diff
@@ -14,36 +14,6 @@
 
 $appId = $_['appId'];
 
-//Gets the current user's language
```

Comment:
Never, ever put logic into your templates. Put this into a controller or a class that handles it. 

If you read it carefully, this is exactly backwards: the template should not fill itself using controller methods, but the controller should render the template by passing in the values.